pkgs/applications/editors/jetbrains/common.nix

```diff
@@ -81,6 +81,7 @@ with stdenv; lib.makeOverridable mkDerivation rec {
         stdenv.cc.cc.lib libsecret e2fsprogs
         libnotify
       ] ++ extraLdPath)}" \
+      ${lib.concatStringsSep " " extraWrapperArgs} \
```

Comment:
Would you expect that those args could override PATH and LD_LIBRARY_PATH but not JDK_HOME, etc?

Reply:
I'm not sure i understand what you mean.

Reply:
PATH and LD_LIBRARY_PATH is prefixed while all the other variables are overriden all the time.

I only realize now that they are not : separated variables. Should be fine I think.